scripts/run_pahfit

```diff
@@ -37,7 +37,9 @@ def initialize_parser():
     parser.add_argument('--saveoutput', action='store',
                         default='ipac', choices=savetypes,
                         help='Save fit results to a file of specified type')
-
+    parser.add_argument('--noguess', action='store_true',
```

Comment:
I don't find "guess" as a very intuitive descriptor.  Like my comment above, I like something like estimate.  With the description text list "Estimate initial starting fit parameter values".

Reply:
I used the word "guess",  because "guess" is what we currently call the method in the code. But I have no preference one the naming as I mentioned above. :)

Reply:
I can rename the "guess" function as well as you proposed in the comment below.

Reply:
Great!